tests/cypress/integration/auth_page.js

```diff
@@ -8,14 +8,15 @@
 
 describe('Check server availability', () => {
   it('Server web interface is available', () => {
-    cy.visit('http://localhost:8080')
+    cy.visit('/')
+    cy.wait(30000)
```

Comment:
What is the usual practice here? Could you please quote its documentation?

Reply:
Cypress documentation about this: https://docs.cypress.io/api/commands/visit.html#Prefixes
I moved "baseUrl" parameter to the configuration file (cypress.json). This will make it easier to navigate in location, and if  need to change it than only need to change it in one place. 
Added "wait" for loading the preloader. This will help avoid errors in checking the availability of page elements when first visit the page after starting docker containers.
This may be reworked in the future when more tests are added.

Reply:
@dvkruchinin 

Hi, maybe try to play with ``visit`` arguments: ``retryOnStatusCodeFailure`` or ``retryOnNetworkFailure`` and ``timeout`` as cypress docs say:

https://docs.cypress.io/api/commands/visit.html#Arguments 

Also from the docs about availability of page elements: 
Can you see what Cypress is doing under the hood? It’s automatically waiting and retrying because it expects the content to eventually be found in the DOM. It doesn’t immediately fail!

https://docs.cypress.io/guides/getting-started/writing-your-first-test.html#Step-2-Query-for-an-element

Reply:
Hi, I removed the wait(). Locally, the test passed without this command.